src/Dapr.Workflow/WorkflowClient.cs

```diff
@@ -0,0 +1,71 @@
+﻿// ------------------------------------------------------------------------
+// Copyright 2022 The Dapr Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//     http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ------------------------------------------------------------------------
+
+using System;
+using System.Threading.Tasks;
+using Microsoft.DurableTask;
+using Microsoft.DurableTask.Client;
+
+namespace Dapr.Workflow
+{
+    // TODO: This will be replaced by the official Dapr Workflow management client.
+    /// <summary>
+    /// Defines client operations for managing Dapr Workflow instances.
+    /// </summary>
+    public sealed class WorkflowClient : IAsyncDisposable
+    {
+        readonly DurableTaskClient innerClient;
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="WorkflowClient"/> class.
+        /// </summary>
+        /// <param name="innerClient">The Durable Task client used to communicate with the Dapr sidecar.</param>
+        /// <exception cref="ArgumentNullException">Thrown if <paramref name="innerClient"/> is <c>null</c>.</exception>
+        public WorkflowClient(DurableTaskClient innerClient)
+        {
+            this.innerClient = innerClient ?? throw new ArgumentNullException(nameof(innerClient));
+        }
+
+        /// <summary>
+        /// Schedules a new workflow instance for execution.
+        /// </summary>
+        public Task<string> ScheduleNewWorkflowAsync(
```

Comment:
Should we also have the param descriptions in this method header similar to how we have done above?

Reply:
Done.